frontend/src/routes/subRoutes/contractRoutes.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Route } from "react-router-dom";
+import ProtectedRoute from "../ProtectedRoute";
+import { PageLink } from "../../components/PageLink";
+import { Contract, Contracts } from "../../pages";
+const contractRoutes = [
+  <Route key="contracts" path="/contracts" element={<ProtectedRoute component={Contracts} />}>
```

Comment:
This is more of a point for discussion, no need to act on this at this time, but given that 99% of things are going to be protected, would it not make more sense to invert the logic, assume everything needs to be protected, and have an exception for the things that don't need protecting?